### PR TITLE
📝 Document custom commands with CLI

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Some people didn't know about the ability to run custom commands with the CLI. See Context.

## What's changed

Added some examples.

### CODE REVIEW

File: docs/src/administration/cli/_index.md [hunk 1]
```diff
@@ -189,3 +189,28 @@ This isn't available by default on macOS, but can be installed via `brew`.
 Check your home directory and ensure that the file `~/.platformsh/autocompletion.sh` is being included by your shell.
 
 If you experience issues, run `platform self:install` to attempt a reinstall of this utility.
+
+### Run commands on your container
+
+You can use the Platform.sh CLI to run commands on your container.
+You can use any command you've added in dependencies or the build hook.
```
Comment (trolologuy) [inline docs/src/administration/cli/_index.md:196]: ⛏️  A link to the dependencies and the build hook might be nice.
- https://cli-ssh-commands-aeihkoy-652soceglkw4u.eu-3.platformsh.site/create-apps/app-reference.html#dependencies
- https://cli-ssh-commands-aeihkoy-652soceglkw4u.eu-3.platformsh.site/create-apps/hooks.html#compile-extension

**Review by trolologuy — APPROVED** [review]
✅ 

- [x] Final built pages are working as expected.
- [x] The information is correct.
- [x] The information is comprehensive.
- [x] The information is concise.
- [x] The information is secure.
- [x] The information is clear.

**Review by trolologuy — APPROVED** [review]
✅

**Comment by CollierCZ** [comment]
Seems like the pre-commit hook took longer than I'd expected and I didn't push the change like I expected :facepalm:

**Review by trolologuy — APPROVED** [review]
✅ Good to go!

**Pull request merged**